docs/_data/table/en/user-configuration.yml

```diff
@@ -31,7 +31,7 @@ alluxio.user.file.master.client.threads:
 alluxio.user.file.metadata.load.type:
   'The behavior of loading metadata from UFS. When information about a path is requested and the path does not exist in Alluxio, metadata can be loaded from the UFS. Valid options are `Always`, `Never`, and `Once`. `Always` will always access UFS to see if the path exists in the UFS. `Never` will never consult the UFS. `Once` will access the UFS the "first" time (according to a cache), but not after that.'
 alluxio.user.file.passive.cache.enabled:
-  'Whether to cache files when reading.'
+  'Whether to cache files to local Alluxio workers when the files are read from remote workers (not UFS).'
```

Comment:
I think this option also applies to read from UFS. When it's on, reading from UFS will also cache to local worker

Reply:
Yeah, so i'd better to make more concrete.

Reply:
did you forget to push the commit for this?

Reply:
@maobaolong  ping?

Reply:
@yupeng9 I've update this PR, PTAL.